CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 0.1.0 - In Progress
```

Comment:
We've released 0.1.0 right? I submitted a PR this morning and made 0.1.1 in progress

Reply:
Yes looks like I messed up the deployment.  The scripts should work but I think I didn't update the CHANGELOG in anticipation of the new scripts properly.

Reply:
Cool. I'll mirror this in my PR.